js/app/components/chat/chat.tsx

```diff
@@ -326,106 +324,106 @@ function ChatMessageRow({
           }
           close();
         }}
-      >
-        <View
-          flexDirection="row"
-          display="block"
-          paddingVertical={isWeb ? 6 : 4} // Adjust padding for web
-          paddingHorizontal={isWeb ? 6 : 4} // Adjust padding for web
-          position="relative"
-          hoverStyle={{ backgroundColor: "rgba(255,255,255,0.1)" }}
-          backgroundColor={
-            currentReplyTo?.cid === message.cid
-              ? "rgba(180,180,255,0.1)"
-              : "transparent"
+      > */}
+      <View
+        flexDirection="row"
+        display="block"
+        paddingVertical={isWeb ? 6 : 4} // Adjust padding for web
+        paddingHorizontal={isWeb ? 6 : 4} // Adjust padding for web
+        position="relative"
+        hoverStyle={{ backgroundColor: "rgba(255,255,255,0.1)" }}
+        backgroundColor={
+          currentReplyTo?.cid === message.cid
+            ? "rgba(180,180,255,0.1)"
+            : "transparent"
+        }
+        borderRadius={isWeb ? 4 : 4}
+        onPress={() => {
```

Comment:
Re-adding the long press to reply-ability to mobile would be good since we're taking away slide to reply
```suggestion
        onLongPress={() => {
            if (!isWeb) {
               handleReply();
            }
          }
        onPress={() => {
```